### DISCUSSION TITLE
Ingame Voting System

### DISCUSSION THREAD

**Discussion by Z1kou** [sunflower-land/sunflower-land#1209] [Ideas]
*We at Pixelized Crib are exploring new features to add to the game, and today I am gonna open several discussion's for every single one of them so sit down get a drink and buckle up because i need your spicy opinion's everyone !*

 # ISSUE

Our Vision is the generalisation and normalisation of crypto and our game, having all age ranges which include non tech savy people that don't know how to fully utilise discord and github and can't voice their opinion

# SOLUTION

Having an ingame voting system for the features that the team need to know players opinion about will be able to let us gather more accurate data as well as reaching the majority of our daily base players and fully connect the player base we strive to have in the future

What are your thoughts Farmers 👨‍🌾 ?

**Comment by Aminbtdds** [comment]
It is a brilliant idea😍

**Comment by ShiningMinions** [comment]
Yes I think this would be such a good feature to have. IMO, moving anything away from Discord and into game seems to almost always be the right move :+1: . I can already picture a new "Goblin Poll Booth" section within goblin town.

(EDIT) I think it's important to also define who this "Voting Booth" is for. I really see this being for the coreDevs. I can see two scenarios:
1. The coreDevs poll certain aspects of new features **before programming them in-game** (A good case to study: old school runescape poll booths).
2. Also, the CoreDevs determine issues in which the community is split. Community will vote here to settle.


## Question:
I am curious - would farmers prefer this to be an on-chain or off-chain thing?

### On-Chain:
- Pros of on-chain is you have guaranteed vote integrity, reliability, and security (being backed by the blockchain). Not even the developers can skew the votes. (NOTE - technically multi-accouters can still skew the vote, but this problem also applies to off-chain).
- Cons of on-chain is that you have to pay a gas fee.

### Off-Chain (perhaps it would work like VIP door where it would vote in Discord upon click or something):
- Pros of off-chain is that you don't pay a gas fee.
- Cons of off-chain is that it's centralized, and there's no guarantee someone in-charge hasn't skewed the results.

## React with one of the following for your opinion:

:+1: = I prefer ON-CHAIN voting

:-1: = I prefer OFF-CHAIN voting

**Reply by future2back** [reply]
> into game seems to almost always be the right move

Thats all.

OFF-CHAIN with public information can be an alternative.

{account} {farm.id} {vote}


Not sure if paying fees would be a problem as they are cheap on this network.
